packages-content-model/roosterjs-content-model-core/lib/editor/StandaloneEditor.ts

```diff
@@ -392,6 +393,20 @@ export class StandaloneEditor implements IStandaloneEditor {
         return this.getCore().trustedHTMLHandler;
     }
 
+    /**
+     * Get the scroll container of the editor
+     */
+    getScrollContainer(): HTMLElement {
+        return this.getCore().domEvent.scrollContainer;
+    }
+
+    /**
+     * Retrieves the rect of the visible viewport of the editor.
+     */
+    getVisibleViewport(): Rect | null {
```

Comment:
I think it is ok to add these 2 functions. You can remove the functions with same names from `ContentModelEditor` now.